test/integration/crud/abstract_operation.test.ts

```diff
@@ -288,26 +339,22 @@ describe('abstract operation', async function () {
       sinon.restore();
     });
 
-    for (const { subclassCreator, subclassType } of subclassArray) {
+    for (const { subclassCreator, subclassType, correctCommandName } of subclassArray) {
       context(`when subclass is ${subclassType.name}`, async function () {
-        it(`subclass prototype's commandName should equal operation.commandName`, async function () {
+        it(`operation.commandName equals correct string`, async function () {
           const subclassInstance = subclassCreator();
-          const prototypeCommandName = Object.getOwnPropertyDescriptor(
-            subclassType.prototype,
-            'commandName'
-          )?.get?.call(null);
-          expect(prototypeCommandName).to.equal(subclassInstance.commandName);
+          expect(correctCommandName).to.equal(subclassInstance.commandName);
```

Comment:
```suggestion
          expect(subclassInstance.commandName).to.equal(correctCommandName);
```

Minor but when this prints an error you'll want it to say the "result" of commandName, did not equal the expected "correctCommandName". The value under test goes into `expect()`